resolwe/flow/tests/test_models.py

```diff
@@ -468,6 +469,7 @@ def test_list(self, os_mock):
         self.assertEqual(self.data.output['file_list'][0]['total_size'], 34)
         self.assertEqual(self.data.output['file_list'][1]['size'], 42000)
         self.assertEqual(self.data.output['file_list'][1]['total_size'], 42042)
+        self.assertEqual(self.data.size, 42076)
```

Comment:
I think it would be better to explicitly write you are comparing `self.data.size` with the sum of file-type outputs' `total_size`es:

```python
self.assertEqual(self.data.size, 34 + 42042)
```